pegasus_wrapper/resource_request.py

```diff
@@ -89,6 +89,7 @@ class SlurmResourceRequest(ResourceRequest):
         default=_DEFAULT_JOB_TIME_IN_MINUTES,
         kw_only=True,
     )
+    exclude_list: Optional[str] = attrib(validator=optional(instance_of(str)), kw_only=True, default=None)
```

Comment:
Can you add a comment explaining how the `exclude_list` should be formatted in the params file?